modules/11-os

```diff
@@ -7,4 +7,4 @@ source "${BASE_DIR}/framework.sh"
 # shellcheck disable=SC1091
 source "/etc/os-release"
 
-print_columns "OS" "${NAME}"
+print_columns "OS" "${NAME} ${VERSION}"
```

Comment:
E.g. Arch Linux doesn't have `VERSION` in `/etc/os-release`, we don't want to fail in that case:
```suggestion
print_columns "OS" "${NAME} ${VERSION:-}"
```